cookbooks/aws-parallelcluster-slurm/templates/default/slurm/login_nodes_daemon_service/login_nodes_daemon_config.json.erb

```diff
@@ -0,0 +1,4 @@
+{
+  "termination_script_path": "<%= node['cluster']['shared_dir'] %>/login_nodes_on_termination_script.sh",
+  "termination_message": "System will be terminated after <%= GRACETIME_PERIOD %> minutes."
```

Comment:
Injected variables must be references with `@`, i.e. `@GRACETIME_PERIOD`

Reply:
Done!